tests/base/src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -483,6 +483,11 @@ protected CallbackHandler getCallbackHandler(String username, String realm, Stri
                 } else if (callback instanceof AuthorizeCallback) {
                     if (token != null) {
                         ((AuthorizeCallback) callback).setAuthorized(true);
+                    } else if(username.equalsIgnoreCase("unauthorizedUser")){
+                        ((AuthorizeCallback) callback).setAuthorized(false);
+                    } else if(username.equals(((AuthorizeCallback) callback).getAuthenticationID()) &&
+                       username.equals(((AuthorizeCallback) callback).getAuthorizationID())) {
+                        ((AuthorizeCallback) callback).setAuthorized(true);
                     } else {
                         if (username.equals(((AuthorizeCallback) callback).getAuthenticationID()) &&
```

Comment:
@darranl This is a minor but these conditions can be simplified, we can merge this now and then I can create a good-first-issue to simplify this

Reply:
IMO I would say go ahead and raise a good first issue - this is not my fix I just addressed a merge conflict so we can clear some of the PR queue backlog.